client/webserver/site/src/js/markets.ts

```diff
@@ -424,7 +423,7 @@ export default class MarketsPage extends BasePage {
     // If the user clicks outside of a form, it should close the page overlay.
     bind(page.forms, 'mousedown', (e: MouseEvent) => {
       if (Doc.isDisplayed(page.vDetailPane) && !Doc.mouseInElement(e, page.vDetailPane)) return this.showVerifyForm()
-      if (!Doc.mouseInElement(e, this.currentForm)) {
+      if (this.forms.currentForm && !Doc.mouseInElement(e, this.forms.currentForm)) {
         closePopups()
       }
     })
```

Comment:
Hmm. Why is this even here? Doesn't the `Forms` class take care of this?

Reply:
Fixed.